src/current/cockroachcloud/advanced-cluster-management.md

```diff
@@ -0,0 +1,188 @@
+    {{site.data.alerts.end}}
+1. In the sidebar, verify the hourly estimated cost for the cluster.
+1. Click **Next: Capacity** and then **Update cluster**.
+
+## Set a maintenance window
+
+From your cluster's [**Overview** page]({% link cockroachcloud/cluster-overview-page.md %}), you can view and manage the maintenance and [patch upgrade]({% link cockroachcloud/upgrade-policy.md %}#patch-version-upgrades) window for your cluster. During the window, your cluster may experience restarts, degraded performance, and downtime for single-node clusters. To help keep your clusters updated while minimizing disruptions, set a window of time when your cluster is experiencing the lowest traffic. If no maintenance window is set, your cluster will be automatically upgraded as soon as new patch versions are available, and other cluster maintenance occurs as needed. Refer to [CockroachDB Cloud Upgrade Policy]({% link cockroachcloud/upgrade-policy.md %}).
+
+{{site.data.alerts.callout_info}}
+Maintenance operations that are critical for cluster security or stability may be applied outside of the maintenance window, and upgrades that begin in a maintenance window may not always be completed by the end of the window.
+{{site.data.alerts.end}}
+
+To set a maintenance window:
+
+1. Click the pencil icon next to **Cluster maintenance** to edit the maintenance window.
+1. From the **Day** dropdown, select the day of the week during which maintenance may be applied.
+1. From the **Start of window** dropdown, select a start time for your maintenance window in UTC.
+
+    The window will last for 6 hours from the start time.
+
+1. (Optional) If you want to delay automatic patch upgrades for 60 days, switch **Delay patch upgrades** to **On**.
+
+    Enable this setting for production clusters to ensure that development and testing clusters are upgraded before production clusters. This setting applies only to patch versions and not to other kinds of upgrades.
+
+## Restore data from a backup
+
+Cockroach Labs runs full backups daily and incremental backups hourly for every CockroachDB {{ site.data.products.advanced }} cluster. Full backups are retained for 30 days and incremental backups for 7 days. See the [Use Managed-Service Backups](use-managed-service-backups.html) page for ways to restore data from your cluster's automatic backups in the Console.
+
+Additionally, you can [back up and restore]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}) your CockroachDB {{ site.data.products.advanced }} cluster manually. For detail on taking backups to your cloud storage, see [Take and Restore Customer-Owned Backups]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}).
+
+{{site.data.alerts.callout_info}}
+All databases are not backed up at the same time. Each database is backed up every hour based on the time of creation. For larger databases, you might see an hourly CPU spike while the database is being backed up.
```

Comment:
What does "based on the time of creation" mean? Is this about the minutes portion of the time the backup happens? It doesn't seem actionable to the customer.